photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -285,8 +284,7 @@ class GameEstimator(val sc: SparkContext, val params: GameParams, implicit val l
       dataSets: Map[String, DataSet[_ <: DataSet[_]]],
       trainingEvaluator: Evaluator,
       validationDataAndEvaluators: Option[(RDD[(Long, GameDatum)], Seq[Evaluator])],
-      normalizationContexts: Option[Map[FeatureShardId, NormalizationContext]])
-    :Seq[(GameModel, Option[EvaluationResults], GameModelOptimizationConfiguration)] = {
+      normalizationContexts: Option[Map[FeatureShardId, NormalizationContext]]): Seq[GameResult] = {
 
     val contextBroadcasts: Option[Map[FeatureShardId, Broadcast[NormalizationContext]]] = normalizationContexts.map {
       contextsMap => contextsMap.mapValues { context => sc.broadcast(context) }
```

Comment:
Yes, this is a rough edge in the implementation currently. What will happen is that we'll have multiple observations that appear to hyperparameter tuning to be identical, but with perhaps different evaluations. This shouldn't break anything, but it's also not ideal. We should probably have a list of lambdas (and elastic net alphas, etc). rather than multiple config specs.